services-manager/pom.xml

```diff
@@ -142,7 +142,7 @@
                                 <replacement>
                                     <token>image.tag</token>
                                     <!--suppress UnresolvedMavenProperty -->
```

Comment:
the readers.app.revision starts with a dash that needs removing to fix the JVM deployment